Upgrade the new microservice build and  modify  the documentation (#2)
khaledk2/ansible-role-omero-ms-image-region: README.md

Role Variables
--------------

- omero_ms_image_region_user: Microservice user, the defaults is 'omero-server'
- omero_ms_image_region_folder: Microservice installation folder
- omero_ms_image_region_port: Microservice port 

> - omero_ms_image_region_db_url: OMERO Database URL

Reviewer: Database -> database (to unify capitalization with the lines below)

--------------

> - omero_ms_image_region_user: 'omero-server': icroservice user, the defaults is omero-server

Reviewer: ```suggestion
- omero_ms_image_region_user: Microservice user, the default is `omero-server`
```

Maybe like this ^^^ ? But double-check please, this particular definition seems to be duplicate one line below and the intention is not clear.

> - omero_ms_image_region_user: 'omero-server'

Reviewer: ```suggestion

```

Maybe remove as it is a duplication ? Please check.

- omero_ms_image_region_update_nginx: if false, it will not update the nginx config file

- omero_ms_image_region_download_URL: The download URl for the distibuted ms build (zip file)
- omero_ms_image_region_ms_sha256: The sha256 for the  for the distibuted ms build

The ms build should be compatible with the installed omero-server, i.e.:

> - The bio-format version for both the microservice and Omero-Server should be the same

Reviewer: ```suggestion
- The Bio-Formats version of the microservice should match the Bio-Formats version of the OMERO.server
```
See 
https://omero.readthedocs.io/en/stable/sysadmins/unix/server-installation.html for motivation of OMERO.server spelling.

> - Also, in case of using ngff data, the version of OMEZarrReader should be the same for both of the Microsevice and the Omero-Server.

Reviewer: Please see https://github.com/khaledk2/ansible-role-omero-ms-image-region/pull/2/files#r1569168184 for OMERO.server spelling - please unify capitalization of "Microservice" (also, here is a typo "Microsevice", "r" missing) in the whole document.

Further spelling suggestions

NGFF (all capitals, it is an abbreviation)

``Also, in case of using`` -> ``If you use`` (drop the "Also" and simplify the "in case of using")

Reviewer: Please try to simplify the formulation as suggested above and use NGFF (not ngff)

- omero_ms_image_region_update_nginx: if false, it will not update the nginx config file

- omero_ms_image_region_download_URL: The download URl for the distibuted ms build (zip file)
- omero_ms_image_region_ms_sha256: The sha256 for the distibuted ms build

The ms build should be compatible with the installed omero-server, i.e.:

> - The bio-format version for the Microservice should match the Bio-Formats version of the Omero-Server

Reviewer: Omero-Server -> OMERO.server
